modin/pandas/dataframe.py

```diff
@@ -11,7 +11,36 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""Module houses ``DataFrame`` class, that is distributed version of ``pandas.DataFrame``."""
+"""
+Module houses ``DataFrame`` class, that is distributed version of ``pandas.DataFrame``.
```

Comment:
Same comments as for base.py. Please check indentation for tables in both files.

Reply:
Changes were added for `dataframe.py` too and intendations checked.